types/node/globals.d.ts

```diff
@@ -71,7 +71,7 @@ declare var module: NodeModule;
 declare var exports: any;
 
 // Buffer class
-type BufferEncoding = "ascii" | "utf8" | "utf-8" | "utf16le" | "ucs2" | "ucs-2" | "base64" | "base64url" | "latin1" | "binary" | "hex";
+type BufferEncoding = "ascii" | "utf8" | "utf-8" | "utf16le" | "ucs2" | "ucs-2" | "base64" | "base64url" | "latin1" | "binary" | "hex" | "base64url";
```

Comment:
```ts
type BufferEncoding = ... | "base64" | "base64url" | ... | "base64url";
```
it's already here,

Reply:
Ah, I missed that 🤦 I think my Node types must have just been out of date. Should have checked that first.

Reply:
No worries, thx for the effort taken!